### PR TITLE
Clean up barrel files and API tags for frontend-authorization

### PR DESCRIPTION
**Pull request by calebmshafer**

### CODE REVIEW

**Review by williamkbentley — APPROVED** [review]

**Review by evelynpreslar-bentley — APPROVED** [review]
looks good to me!

**Comment by calebmshafer** [comment]
@Mergifyio update

File: clients/frontend-authorization/src/frontend-authorization-client.ts [hunk 1]
```diff
@@ -3,9 +3,13 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 export * from "./FrontendAuthorizationClient";
+export * from "./FrontendAuthorizationClientLoggerCategory";
```
Comment (aruniverse) [inline clients/frontend-authorization/src/frontend-authorization-client.ts:6]: thanks! just hit this myself. 

will this be backported to 2.19?

Comment (calebmshafer) [reply clients/frontend-authorization/src/frontend-authorization-client.ts:6]: That makes sense. I'll backport it.

**Comment by calebmshafer** [comment]
@Mergifyio backport release/2.19.x

**Comment by calebmshafer** [comment]
@skirby1996 FYI we should include this in the patch if we can.  I anticipate another one soon too so we could always hold this until that one.

**Comment by calebmshafer** [comment]
@Mergifyio update

**Comment by calebmshafer** [comment]
@Mergifyio update

**Pull request merged**